odoo/addons/base/models/ir_mail_server.py

```diff
@@ -378,6 +378,12 @@ def action_retrieve_max_email_size(self):
         self.ensure_one()
         return self.test_smtp_connection(autodetect_max_email_size=True)
 
+    @classmethod
+    def _disable_send(cls):
+        """Whether to disable sending e-mails"""
+        # no e-mails during testing or when registry is not ready
+        return modules.module.current_test or not cls.pool._init
```

Comment:
I poll._init is here because it was there above but this seems to read like "disable sending if we're testing or if we don't need to initialize the pool", so we disable sending most of the time. Seems like it should be the other way around (?)

Reply:
Disable sending when: "we are testing" or "registry is ***not*** initializing" (so we don't send e-mails during installation).
The condition is the same as before, it was written in the `mail` module.

Reply:
"Disable when registry is not initializing" is the issue, in my reasoning. It should be disabled when initializing.
I did see it was like that in mail, but mail isn't without fault..

I left this comment because currently on master mail sending is broken, and inverting this condition seemed to fix it.

Reply:
#199659